dart/constraint/ContactConstraint.cpp

```diff
@@ -114,14 +114,21 @@ ContactConstraint::ContactConstraint(
   //----------------------------------------------
   // TODO(JS): Assume the frictional coefficient can be changed during
   //           simulation steps.
-  // Update mFrictionalCoff
+  // Update mFrictionCoeff
   const double frictionCoeffA = computeFrictionCoefficient(shapeNodeA);
```

Comment:
```suggestion
  const double primaryFrictionCoeffA = computeFrictionCoefficient(shapeNodeA);
```

Nitpick: I know it'll make for a bigger diff, but I think I'd recommend adding `primary` to these variable names for better symmetry.

Reply:
updated variable names in 2a0934f, and added the `computePrimaryFrictionCoefficient` function in 760cf3b